### PR TITLE
KOGITO-4336 - Replace CDI qualifier with Quarkus build property

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

File: trusty/trusty-service/trusty-service-common/pom.xml [hunk 1]
```diff
@@ -106,20 +106,6 @@
   </dependencies>
   <build>
     <plugins>
-      <plugin>
-        <groupId>io.quarkus</groupId>
```
Comment (cristianonicolai) [inline trusty/trusty-service/trusty-service-common/pom.xml:110]: @r00ta @danielezonca this one came up during the change, Im not sure if trusty-service-common is intended to be a Quarkus dist app or simply a common module that is reused, which in that case the Quarkus build part can be removed. WDYT?

Comment (danielezonca) [reply trusty/trusty-service/trusty-service-common/pom.xml:110]: Yes I agree I think it is a leftover after one of past refactoring 🤔

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Review by radtriste — APPROVED** [review]
code lgtm

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Review by danielezonca — APPROVED** [review]

**Pull request merged**